core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -0,0 +1,410 @@
+          "allMaxMemory" : allMaxMemory, "allDiskUsed" : allDiskUsed, "allTotalCores" : allTotalCores, "allMaxTasks" : allMaxTasks,
+          "allActiveTasks" : allActiveTasks, "allFailedTasks" : allFailedTasks, "allCompletedTasks" :
+          allCompletedTasks, "allTotalTasks" : allTotalTasks, "allTotalDuration" : allTotalDuration,
+          "allTotalGCTime" : allTotalGCTime, "allTotalInputBytes" : allTotalInputBytes,
+          "allTotalShuffleRead" : allTotalShuffleRead, "allTotalShuffleWrite" : allTotalShuffleWrite };
+      var activeSummary = { "execCnt" : ( "Active(" + activeExecCnt + ")"), "allRDDBlocks" : activeRDDBlocks, "allMemoryUsed" : activeMemoryUsed,
+            "allMaxMemory" : activeMaxMemory, "allDiskUsed" : activeDiskUsed, "allTotalCores" : activeTotalCores, "allMaxTasks" : activeMaxTasks,
+            "allActiveTasks" : activeActiveTasks, "allFailedTasks" : activeFailedTasks, "allCompletedTasks" :
+          activeCompletedTasks, "allTotalTasks" : activeTotalTasks, "allTotalDuration" : activeTotalDuration,
+            "allTotalGCTime" : activeTotalGCTime, "allTotalInputBytes" : activeTotalInputBytes,
+            "allTotalShuffleRead" : activeTotalShuffleRead, "allTotalShuffleWrite" : activeTotalShuffleWrite };
+      var deadSummary = { "execCnt" : ( "Dead(" + deadExecCnt + ")" ), "allRDDBlocks" : deadRDDBlocks, "allMemoryUsed" : deadMemoryUsed,
+            "allMaxMemory" : deadMaxMemory, "allDiskUsed" : deadDiskUsed, "allTotalCores" : deadTotalCores, "allMaxTasks" : deadMaxTasks,
+            "allActiveTasks" : deadActiveTasks, "allFailedTasks" : deadFailedTasks, "allCompletedTasks" :
+            deadCompletedTasks, "allTotalTasks" : deadTotalTasks, "allTotalDuration" : deadTotalDuration,
+            "allTotalGCTime" : deadTotalGCTime, "allTotalInputBytes" : deadTotalInputBytes,
+            "allTotalShuffleRead" : deadTotalShuffleRead, "allTotalShuffleWrite" : deadTotalShuffleWrite };
+
+      var data = {executors: response, "execSummary": [activeSummary, deadSummary, totalSummary]};
+      $.get(createTemplateURI(), function(template) {
+
+        executorsSummary.append(Mustache.render($(template).filter("#executors-summary-template").html(),data));
+        var selector = "#active-executors-table";
+        var conf = {
+                    "data": response,
+                    "columns": [
+                        //{data: 'id'},
+                        {data: function(row, type) { return type !== 'display' ? (isNaN(row.id) ? 0 : row.id ) : row.id;} },
+                        {data: 'hostPort'},
+                        {data: 'isActive', render: formatStatus },
+                        {data: 'rddBlocks'},
+                        {data: function(row, type) { return type === 'display' ? (formatBytes(row.memoryUsed, type) + ' / ' + formatBytes(row.maxMemory, type)) : row.memoryUsed; } },
```

Comment:
this is showing up with 3 digits after .
39.266 KB / 384.093 MB

previously was using Utils.bytesToString which made it 1 digit after . which I think is more readable so can we change to that